setup.py

```diff
@@ -58,6 +58,13 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
     ],
+    entry_points={
+        'console_scripts': [
+            'cling_kernel=clingkernel:main',
```

Comment:
Let's not add these. This can all be done more flexibly with arguments to a single entrypoint, rather than creating a bunch of entrypoints.